Libraries/Components/Button.windows.js

```diff
@@ -116,6 +143,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    var content =
```

Comment:
this assignment should be in an else, or removed altogether along with the OS check since this file is already *.windows.js.

Reply:
I followed the pattern that seemed to be in place for the case where upstream implementation is common (i.e. one file). According to my understanding this makes carrying over upstream changes easier (since the diff's are easier to understand), and allows for eventually moving all upstream.
I don't care one way or another, we could change this, but I guess it'll have to happen separately, consistently throughout the whole codebase (Text, ScrollView, etc. are in same situation).

Reply:
So I renamed to FocusableWindows.
It'll become Focusable if/when the full name refactoring happens.

Reply:
ok, then please submit a separate PR that merges the upstream JS changes. we're already behind if that's the strategy.